registrations/tests/test_signup_group_patch.py

```diff
@@ -33,202 +41,312 @@ def assert_patch_signup_group(api_client, signup_group_pk, signup_group_data):
 
 @freeze_time("2023-03-14 03:30:00+02:00")
 @pytest.mark.django_db
-def test_registration_admin_can_patch_signup_group_extra_info(
-    user_api_client, registration, user
-):
-    user.get_default_organization().registration_admin_users.add(user)
+def test_registration_admin_can_patch_signup_group_extra_info(api_client, event):
```

Comment:
`test_regular_user_cannot_patch_signup_group_extra_info`, `test_admin_cannot_patch_signup_group_extra_info` and `test_registration_admin_can_patch_signup_group_extra_info` are all quite similar. Maybe you could have a common test case for them all.
e.g.
```
@pytest.mark.parametrize(
    "user_role",
    [
        "admin",
        "registration_admin",
        "regular",
        "superuser",
    ],
)
@freeze_time("2023-03-14 03:30:00+02:00")
@pytest.mark.django_db
def test_user_tries_to_patch_signup_group_extra_info(api_client, event, user_role):
    if user_role == "admin":
        user = UserFactory()
        user.admin_organizations.add(event.publisher)
    elif user_role == "registration_admin":
        user = UserFactory()
        user.registration_admin_organizations.add(event.publisher)
    elif user_role == "regular":
        user = UserFactory()
        user.organization_memberships.add(event.publisher)
    else:
        user = UserFactory(is_superuser=True)

    registration = RegistrationFactory(event=event)

    signup_group = SignUpGroupFactory(registration=registration)
    first_signup = SignUpFactory(
        signup_group=signup_group,
        registration=registration,
        extra_info="signup1 extra info",
    )
    second_signup = SignUpFactory(
        signup_group=signup_group,
        registration=registration,
        extra_info="signup2 extra info",
    )

    signup_group_data = {"extra_info": "signup group extra info"}

    assert signup_group.extra_info is None
    assert first_signup.extra_info == "signup1 extra info"
    assert second_signup.extra_info == "signup2 extra info"

    api_client.force_authenticate(user)

    #  Registration admins and superadmin users can patch signup group
    if user_role in ["registration_admin", "superuser"]:
        response = assert_patch_signup_group(api_client, signup_group.id, signup_group_data)
        assert response.data["extra_info"] == signup_group_data["extra_info"]
    #  Regular users and admins cannot patch signup group
    else:
        response = patch_signup_group(api_client, signup_group.id, signup_group_data)
        assert response.status_code == status.HTTP_403_FORBIDDEN

    signup_group.refresh_from_db()
    first_signup.refresh_from_db()
    second_signup.refresh_from_db()
    #  Registration admins and superadmin users can patch signup group
    if user_role in ["registration_admin", "superuser"]:
        assert signup_group.extra_info == signup_group_data["extra_info"]
    #  Regular users and admins cannot patch signup group
    else:
        assert signup_group.extra_info == None
    assert first_signup.extra_info == "signup1 extra info"
    assert second_signup.extra_info == "signup2 extra info"
```